enterprise/views.py

```diff
@@ -106,12 +122,54 @@ def get_note(provider, required):
                 " not be able to get any discounts from {provider}."
             ).format(provider=provider)
 
-    def get(self, request):
+    @staticmethod
+    def get_course_note(provider, course_name):
         """
-        Render a form to collect user input about data sharing consent.
+        Get a course-specific note for the form.
+
+        Arguments:
+            provider: The name of the linked EnterpriseCustomer
+            course_name: The name of the course in question
+        """
+        return _(
+            "Data sharing consent is required for courses provided by {provider}, including "
+            "{course_name}; if consent is not provided, you will be returned to the Dashboard."
```

Comment:
Courses from {provider} require data sharing consent. If you do not agree to share your data, you will be redirected to your dashboard.